doc/api/child_process.md

```diff
@@ -849,6 +849,10 @@ added: v0.5.9
 The `'message'` event is triggered when a child process uses [`process.send()`][]
 to send messages.
 
+*Note*: since the message goes through JSON serialization and parsing, it might
+not be the same as what is originally sent, see notes of
+[the `JSON.stringify()` specification][`JSON.stringify` spec].
```

Comment:
Perhaps simplify the link and adjust some of the wording here... for example:

<pre>
*Note*: since the message is transmitted as JSON, not all values may be transferred as-is. See the documentation for [`JSON.stringify()`][] for more details.
</pre>

Where the JSON.stringify() link goes to MDN instead (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#Description), which IMHO has a more readable description of when/how values are converted.